core/client/fs/src/main/java/alluxio/client/file/cache/CacheManager.java

```diff
@@ -100,21 +100,29 @@ public static CacheManager get(AlluxioConfiguration conf) throws IOException {
 
     /**
      * @param conf the Alluxio configuration
-     * @param pageMetaStore meta store for pages
+     * @return an instance of {@link CacheManager}
+     */
+    public static CacheManager create(AlluxioConfiguration conf) throws IOException {
+      return create(conf, CacheManagerOptions.create(conf));
+    }
+
+    /**
+     * @param conf the Alluxio configuration
+     * @param options the options for local cache manager
      * @return an instance of {@link CacheManager}
      */
     public static CacheManager create(AlluxioConfiguration conf,
```

Comment:
why need to pass conf here, can we directly pass cachemanagerOptions only?

Reply:
good catch, because the shadow cache manager is still using the conf, I will post another pr just for shadow cache, then we could totally get rid of conf.